### ISSUE TITLE
Warning in net6 preview

### ISSUE THREAD

**Issue by newbe36524** [mingyaulee/Blazor.BrowserExtension#6]
1. create a project
2. add Blazor.BrowserExtensions
3. create mainfest.json
4. build

There is a warning and I have no idea what it is about.

```
Blazor.BrowserExtension.Build.targets(145, 5): Unable to find text to replace 'this.mono_wasm_assembly_already_added' in file 'C:\Repos\newbe\Newbe.Demo\src\BlogDemos\Newbe.Blazor\Newbe.Blazor.Template\Newbe.Blazor.Template\bin\Debug\net6.0\wwwroot\framework\dotnet.6.0.0-preview.3.21201.4.js'
```

And it seem that everything works well event the warning raised

**Comment by mingyaulee** [comment]
Yeah the reason for this is because I have yet to test it out with .Net 6. Currently the plan is to support .Net 6 once it is out of i, but it's good that you have already tried it out and it works (if you ignore the warning).

**Comment by mingyaulee** [comment]
Fixed with v0.1.12
